src/Umbraco.Cms.StaticAssets/wwwroot/App_Plugins/Umbraco.BlockGridEditor.DefaultCustomViews/umbBlockGridDemoRichTextBlock.html

```diff
@@ -21,5 +21,5 @@
 
 </style>
 
-<div class="text" ng-click="block.edit()" ng-focus="block.focus" ng-bind-html="block.data.richText" style="margin: 0 20px;">
+<div class="text" ng-click="block.edit()" ng-focus="block.focus" ng-bind-html="sce.trustAsHtml(block.data.richText)" style="margin: 0 20px;">
```

Comment:
It could just use the `safe_html` filter instead and not expose the `$sce` object.

```
 ng-bind-html="block.data.richText | safe_html"
```

Reply:
You're right, didn't spot that filter. I've reverted the changes to the component and updated the custom view